src/fragments/lib/auth/android/signin_next_steps/80_current_user.mdx

```diff
@@ -0,0 +1,51 @@
+This call fetches the current logged in user.
```

Comment:
It would also help customers to know when the API would return a user vs `null`, what values does the result contain and which API calls would result in a user not being null.